conda.recipe/meta.yaml

```diff
@@ -1,30 +1,25 @@
-{% set name = "conda-package-handling" %}
-{% set version_match = load_file_regex(
-  load_file="src/conda_package_handling/__init__.py",
-  regex_pattern='^__version__ = "(.+)"') %}
-{% set version = version_match[1] %}
-
 package:
-  name: {{ name }}
-  version: {{ version }}
+  name: conda-package-handling
+  version: version: {{ GIT_DESCRIBE_TAG }}+{{ GIT_BUILD_STR }}
```

Comment:
Version version?

Reply:
Oh. Will fix

Reply:
It looks like this expands to `+`only on the py37 test.